ansible/roles/nginx/vars/couchdb2.yml

```diff
@@ -21,3 +21,4 @@ nginx_sites:
        balancer: "{{ deploy_env }}_couchdb_{{ couchdb2_proxy_port }}"
        proxy_redirect: ' off'
        proxy_buffering: ' off'
+       proxy_read_timeout: ' 65s'
```

Comment:
we need this to be slightly more than 60s or change the heartbeat for the changes feed. I figured this was easier.

The default heartbeat and proxy_read_timeout are both 60s so it timesout just as the heartbeat is sent